peer_test.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/roasbeef/btcutil"
 )
 
-func disablePeerLogger(t *testing.T) {
+func init() {
```

Comment:
Will this _only_ run when the tests are executing? Or will this run unconditionally (meaning this'll disable logging when `lnd` is running)?

Reply:
afaia, it's only called when compiling a test executable since it's in a `_test.go` file